src/Gateway/MolliePaymentGateway.php

```diff
@@ -839,7 +839,7 @@ public function process_refund($order_id, $amount = null, $reason = '')
         );
 
         // If there is no Mollie Payment Order object, try getting a Mollie Payment Payment object
-        if ($payment_object_id === null) {
+        if (empty($payment_object_id)) {
```

Comment:
Hint: There is no need for use `empty()`  you can simply use  `if (! $payment_object_id) {`.  `empty()` is only important when maybe the variable is not declared.